codebuild_specs/run_coverage_linux.yml

```diff
@@ -0,0 +1,30 @@
+version: 0.2
+env:
+  shell: bash
+  variables:
+    CLI_REGION: us-east-1
+    E2E_TEST_COVERAGE_DIR: node_v8_coverage
+  secrets-manager:
+    CODECOV_TOKEN: arn:aws:secretsmanager:us-east-1:052112422291:secret:testing/codecov/token-OB8isC
```

Comment:
We should not have Secrets Manager ARNs on Github. This should be an env variable that is injected in